packages/eos-components/src/components/GradeSelectionModal.vue

```diff
@@ -0,0 +1,134 @@
+<template>
+
+  <div>
+    <b-modal
+      id="grade-selection-modal"
+      size="xl"
+      centered
+      :visible="visible"
+      :noCloseOnBackdrop="true"
+      :noCloseOnEsc="true"
+      :hideFooter="true"
+      :hideHeaderClose="true"
+    >
+      <b-container>
+        <h1 class="text-primary">
+          Select grade range
+        </h1>
+        <h6 class="text-muted">
+          The Endless Key download for ChromeOS contains a sample of the
+          content available on the full Endless Key. The content is available
+          in three selections:
+        </h6>
+
+        <b-card-group deck class="py-5">
+          <b-card
+            v-for="grade in grades"
+            :key="grade.key"
+            :title="grade.key"
+            :borderVariant="selection === grade.key ? 'primary' : 'default'"
+            titleTag="h6"
+            subTitleTag="h4"
+            class="welcome-card"
```

Comment:
Since there are no other cards in this component you can just use the `.card` CSS class for referencing it. Thus this line is not needed.

Reply:
The `.welcome-card` styles are defined in [the styles](https://github.com/endlessm/kolibri-explore-plugin/pull/439/files#diff-3d1fd9dd539be4bf82ce6e5adb1a46c94d63b83ba763f2f5e5c71c9e260e67d1R98) that's why this is used.